### DISCUSSION TITLE
New optional feature request: Bookmark's folder name in search results

### DISCUSSION THREAD

**Discussion by Hept0p** [aaFn/Bookmark-search-plus-2#309] [Q&A]
some bookmark saved this way:
-folder1: Home supply (folder name)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-bookmark1: CASA - casa39.com  (bookmark name)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-bookmark2:  GUIN - guin38.com (bookmark name)
  
Searching for "home" will output no reasult because BSP2 doesn't look for folder name by default.
Having an option to allows this would be useful.

**Comment by aaFn** [comment]
Hello @Hept0p , BSP2 does look for folder names in the search by default.
If it is not happening on your side, that means you disabled it at some time, by clicking on the magnifier glass button, and by selecting "Show bookmarks only".
To restore it, simply click on the "Reset filters" button, or click again on the magnifying glass to re-enable it.

**Comment by aaFn** [comment]
Closing the discussion, since no follow up
